modules/local/hifiadapterfilt/tests/main.nf.test

```diff
@@ -0,0 +1,54 @@
+nextflow_process {
+
+    name "Test Process HIFIADAPTERFILT"
+    script "../main.nf"
+    process "HIFIADAPTERFILT"
+
+    tag "modules"
+    tag "modules_ebimetagenomics"
+    tag "hifiadapterfilt"
+
+    test("hifiadapterfilt - fastq.gz") {
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ],
+                    file("${moduleDir}/tests/fixtures/SRR30463389_subreads.fastq.gz", checkIfExists: true)
```

Comment:
Have you tried finding test dataset in nf-core repo? Something like https://github.com/nf-core/test-datasets/tree/genomeassembler/A_thaliana_Col-0_2mb
You can use it with link instead of adding fq.gz file into repo
like here https://github.com/EBI-Metagenomics/nf-modules/blob/main/tests/modules/ebi-metagenomics/prodigal/main.nf or there are many examples in nf-core repo